recipes/pathogen-embed/meta.yaml

```diff
@@ -1,15 +1,16 @@
 {% set version = "2.2.0" %}
+{% set name = "pathogen-embed" %}
 
 package:
-  name: pathogen-embed
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/9c/96/971af8ac8519e8924624f70896c73d0d8a9491ee76095391c4da216d4aa4/pathogen_embed-{{ version }}.tar.gz
```

Comment:
In another PR I noticed that the bot could update even pythonhosted urls! Amazing!

Reply:
Possibly - but I'm not sure it works 100% reliably?

It's definitely better to use the predictable URLs as for manual updating then only the sha needs to be changed.

Also, it keeps history/blame cleaner if we don't change the URL with every version bump.